pkg/controllers/state/statenode.go

```diff
@@ -195,9 +195,6 @@ func (in *StateNode) ValidateNodeDisruptable(ctx context.Context, kubeClient cli
 	}
 	// check whether the node has all the labels we need
 	for _, label := range []string{
```

Comment:
nit: You almost don't need to use a `for` loop anymore if you are just checking a single key